### ISSUE TITLE
Chromecast receiver with custom header

### ISSUE THREAD

**Issue by fridapinto-debug** [google/ExoPlayer#8642]
Hello I have created the chromecast receiver correctly and I can cast dash , mp4 and hls videos correctly. I also implemented drm license request using licenserequesthandler.

I want to know how can I add custom headers to manifestrequest and segment request? I want to check specific headers for media to be streamed like custom User-Agent, adding extra header.

so far I have done:  

```
playbackConfig.manifestRequestHandler = requestInfo => {
--
 // requestInfo.withCredentials = true;
  requestInfo.headers['User-Agent'] = 'Mozilla 6.0 firefox';
  
  
   };
```

but still the useragent is same when the chromecast calls the url.

**Comment by AquilesCanta** [comment]
If I understand the question correctly: You want to set the user agent from javascript in your receiver app, then this falls outside of the scope of the ExoPlayer issue tracker. You might want to try your luck asking in the [Cast's issue tracker](https://github.com/googlecast/CastReceiver/issues). I am not sure whether this is supported by chrome in general, maybe @marcbaechinger will know. Marc, any pointers? Otherwise, I think it's ok to close since there's not much we can do here.

**Comment by fridapinto-debug** [comment]
@AquilesCanta  you are right I just wanted to have a custom header on the requested media.
